Align usage report config with Grafana (#5406)
grafana/loki: docs/sources/configuration/_index.md

[ring: <ring>]
```

## analytics

> This block allow to configure usage report of Loki to grafana.com

Reviewer: ```suggestion
This block allows to configure analytics reporting of Loki to grafana.com
```

> This block allows to configure analytics reporting of Loki to grafana.com

Reviewer 2: For future writing tasks, the sentence construction "allows to configure" (allows to something) is not proper English grammar. Instead of this, leave out the "allows to" phrase, and use the present tense of the verb ("configure" in this case).
 
```suggestion
The `analytics` block configures the reporting of Loki analytics to grafana.com
```

```yaml

> # Whether or not usage report should be disabled.

Reviewer 2: Please change the description to match the new parameter.
```suggestion
# When true, enables usage reporting.
```
